### PR TITLE
command to print total number of nodes

### PR DESCRIPTION
**Pull request by stan-confluent**
```shell
 ducktape systests/cluster/test_remote_account.py::TestClusterSpec --total-size --collect-only 2>/dev/null
5
```

### CODE REVIEW

**Review by imcdo — COMMENTED** [review]
Why have a seperate flag thats part of collect only, it seems wierd to have the dependancy, why not just have:
```
ducktape tests --collect-num-nodes ... 
``` 
instead

File: ducktape/command_line/main.py [hunk 1]
```diff
@@ -165,6 +165,11 @@ def main():
             print("    " + str(test))
         sys.exit(0)
 
+    if args_dict["collect_num_nodes"]:
+        total_nodes = sum(test.expected_cluster_spec.size() for test in tests)
```
Comment (abolipai99) [inline ducktape/command_line/main.py:169]: We can use test.expected_num_nodes here. It's returns the same value right now but would be helpful if there's any change in it in the future.

Comment (stan-confluent) [reply ducktape/command_line/main.py:169]: good idea, changed

**Review by imcdo — APPROVED** [review]

**Comment by imcdo** [comment]
@stan-confluent is there anything blocking this from being merged?

**Pull request merged**